tests/unit/scanners/test.javascript.js

```diff
@@ -352,114 +290,34 @@ describe('JavaScript Scanner', () => {
         fakeMetadata
       );
 
-      const { linterMessages } = await jsScanner.scan();
+      const { linterMessages } = await runJsScanner(jsScanner);
       expect(linterMessages.length).toEqual(0);
     });
   });
 
+  // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Same as previous comment, this eslint-disable-next-line doesn't seem to be needed anymore.

Reply:
oops, sorry